src/web/mjs/connectors/AssortedScans.mjs

```diff
@@ -0,0 +1,65 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs'
+
+export default class AssortedScans extends Connector {
+    constructor() {
+        super();
+        super.id = 'assortedscans';
+        super.label = 'assortedscans';
+        this.tags = ['manga', 'english'];
+        this.url = 'https://assortedscans.com/';
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '#series-title');
+        let title = data.text.trim();
+        let id = uri.pathname + uri.search;
+        return Manga(this, id, title);
+    }
+
+    async _getMangas() {
+        let request = new Request(new URL('/reader/', this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'section.series h2.series-title a');
+        return data.map(element => {
+                return {
+                    id: element.href,
+                    title: element.text.trim(),
+                };
+            }
+        )
+    }
+
+    async _getChapters(manga) {
+        let id = manga.id.match('/reader/.*/?')[0];
+        let request = new Request(new URL(id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'div.chapter a');
+        return data
+            .filter((_, i) => (i + 1) % 2) // Every second chapter link is a link to the translator
+            .map(element => {
+                return {
+                    id: element.href,
+                    title: element.title
+                }
+            });
+    }
+
+    async _getMaxSite(chapter) {
+        let id = chapter.id.match('/reader/.*/?')[0];
+        let request = new Request(new URL(id + '1/', this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'li.dropdown-element.page-details:last-child a');
+        let maxSite = data['0'].text.match(/Page (\d+)/)[1];
+        return [id, parseInt(maxSite)];
+    }
+
+    async _getPages(chapter) {
```

Comment:
Do not scrape the images if they are paginated, use the `handleConnectorURI` mechanism instead by returning the list of HTML pages containing the images and then define a handler that will extract the image for a given HTML page.

An outdated sample can be found for [AnimExtremist](https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/AnimExtremist.mjs#L59-L92)